integrationtests/src/test/resources/log4j2-test.properties

```diff
@@ -43,13 +43,13 @@ rootLogger.additivity = false
 #logger.netty.additivity = false
 #logger.netty.appenderRef.console.ref = STDOUT
 #
-logger.broker.name = kafka
-logger.broker.level = DEBUG
-logger.broker.additivity = false
-logger.broker.appenderRef.console.ref = STDOUT
-
-logger.net.name = org.apache.kafka.clients.NetworkClient
-logger.net.level = debug
+#logger.broker.name = kafka
```

Comment:
Did you mean to turn the noise down here too?

Reply:
It looks like someone (me - 7d673aee49b3a4c62a5ca8902c680a7c05987341 ) had left those loggers enabled after the multitenant work. That wasn't intentional.

The build was so noisy we were missing stack traces such as the ones that provoked this issue.